PD-1460 / 25.04 / PD-1460-replication-task-status-reword (#3489)
truenas/documentation: content/SCALE/SCALEUIReference/DataProtection/ReplicationScreensSCALE.md

| Setting | Description |
|---------|-------------|
| **Destination** | (Required) Enter or use <span class="material-icons">arrow_right</span> to the left of the **/mnt** folder and at each dataset to expand the dataset tree to browse to the dataset location that has snapshots to replicate. Click on the dataset or directory name, folder icon, or checkbox to select the dataset or directory. Selecting a location defines the full path to that location as the destination. Appending a name to the path creates a new zvol at that location. For example, selecting *pol1/dataset1* stores snapshots in *dataset 1*, but adding */zvol1* after *dataset*1 creates *zvol1* for snapshot storage. Click the <span class="material-icons"><span class="material-icons">arrow_drop_down</span></span> at the **/mnt** folder to collapse the dataset tree. |

> | **Destination Dataset Read-Only Policy** | Select the policy from the dropdown list. Options:<br><li>**Set** that changes all destination datasets to `readonly=on` after a successful replication.<br><li> **Require** stops replication unless all existing destination datasets have the property `readonly=on`.<br><li> **Ignore** disables checking the `readonly` property during replication.</li> |

Reviewer: We should not insert code properties unless the user needs to interact with it or sees it in the UI, translate these mentions into simple explanations or descriptions is more user friendly. For example: 
```suggestion
| **Destination Dataset Read-Only Policy** | Select the policy from the dropdown list. Options:<br><li>**Set** - Select to change all destination datasets to readonly after a successful replication.<br><li> **Require** - Select to stop replication unless all existing destination datasets are readonly.<br><li> **Ignore** - Select to disable checking readonly during replication.</li> |
```
We need to think about how Global English applies as those users might not understand what we mean and Google Translate might not help much with this content.
While I'm not a fan of saying successful,  it is not incorrect or misleading. :-)
